phoenix-core/pom.xml

```diff
@@ -194,6 +194,24 @@
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.xolstice.maven.plugins</groupId>
+        <artifactId>protobuf-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>compile-protoc</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>compile</goal>
+            </goals>
+            <configuration>
+              <protoSourceRoot>${basedir}/src/main/protobuf/</protoSourceRoot>
```

Comment:
These are redundant with  the parent dependencyManagement section.
Setting them here, or in the dependencyManagement section should be enough.

Reply:
Sorry, I only meant the configuration entries.
Declaring the plugin and the version in the top dependencyManagement section is good practice.

Reply:
Kept version declaration property in parent pom and defined entire plugin in phoenix-core, but if declaring plugin with version on parent pom is good practice, let me change it.